rewrite-java/src/main/java/org/openrewrite/java/cleanup/UseObjectNotifyAll.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.search.UsesMethod;
+import org.openrewrite.java.tree.J;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * Recipe to use {@link Object#notifyAll()} instead of {@link Object#notify()}
```

Comment:
All of this context in this javadoc is great, and I'd love to see it move down into the `getDescription()` on the recipe so that people see all of it when using the recipe.

Reply:
Thank you for the feedback @jkschneider. I extracted them with d7d4c9ee1288d1880d20d99c02e88e5445a981f8